File: Java/Ex018.java
/*
Ex018 Faça um programa que leia um ângulo qualquer e mostre na tela o 
valor do seno, cosseno e tangente desse ângulo.
 */
package ex018;

import java.util.Scanner;

public class Ex018 {


    public static void main(String[] args) {
        Scanner t = new Scanner(System.in);
        System.out.println("Digite um ângulo: ");
        double num = t.nextDouble();
        double a = Math.toRadians(num);

        double se = Math.sin(a);
        double co = Math.cos(a);
        double ta = Math.tan(a);
        float b = (float)se;
        float c = (float)co;
        float d = (float)ta;
        System.out.printf("SENO: %.2f \n", b);
        System.out.printf("COSSENO: %.2f \n", c);
        System.out.printf("TANGENTE: %.2f \n", ta);
    
    }
    
}

File: Python/ex018.py
#Ex018 Faça um programa que leia um ângulo qualquer e mostre na tela o valor do seno, cosseno e tangente desse ângulo.

from math import sin, radians, cos, tan
a = float(input('Digite o ângulo que você deseja: '))
seno = sin(radians(a))
cos = cos(radians(a))
tan = tan(radians(a))
print(f'O angulo {a} tem o SENO de {seno:.2f}')
print(f'O ângulo {a} tem o COSSENO de {cos:.2f}')
print(f'O ângulo {a} tem o TANGENTE de {tan:.2f}')
